### PR TITLE
fix: rest api contract deployment issue

### PR DESCRIPTION
**Pull request by solarw**
Addresses #262

### CODE REVIEW

**Comment by 5A11** [comment]
This is blocked. @MissingNO57 can you have a look at this please?

File: cosmpy/common/rest_client.py [hunk 1]
```diff
@@ -109,6 +110,14 @@ def post(self, url_base_path: str, request: Message) -> bytes:
         """
         json_request = MessageToDict(request)
 
+        # Workaround
```
Comment (solarw) [inline cosmpy/common/rest_client.py:113]: good, seems it's a proper solution, cause provided data in bad format

**Comment by solarw** [comment]
@5A11 looks like workaround from @MissingNO57  is a good way to solve this issue.

assume it's ready to be merged in

File: tests/integration/test_contract.py [hunk 2]
```diff
@@ -128,5 +132,19 @@ def test_contract_schema_validation(self):
             raise ValidationTestFailure("Msg should have failed validation") from exc
 
 
+class TestContractRestAPI(TestContract):
```
Comment (5A11) [inline tests/integration/test_contract.py:135]: Why not just use `NetworkConfig.fetchai_stable_testnet`?
Also in the other test file.

Comment (solarw) [reply tests/integration/test_contract.py:135]: it's grpc, but rest is needed here

**Pull request merged**